python/lsst/obs/fiberspectrograph/translator.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import os
+
+import astropy.units as u
+from astropy.time import Time
+
+from astro_metadata_translator import cache_translation
+from lsst.obs.lsst.translators.lsst import SIMONYI_TELESCOPE, LsstBaseTranslator
+
+from lsst.utils import getPackageDir
+
+__all__ = ["FiberSpectrographTranslator", ]
+
+log = logging.getLogger(__name__)
+
+
+class FiberSpectrographTranslator(LsstBaseTranslator):
+    """Metadata translator for Rubin calibration fibre spectrographs headers"""
+
+    name = "FiberSpectrograph"
+    """Name of this translation class"""
+
+    supported_instrument = "FiberSpec"
+    """Supports the Rubin calibration fibre spectrographs."""
+
+    default_search_path = os.path.join(getPackageDir("obs_fiberspectrograph"), "corrections")
+    """Default search path to use to locate header correction files."""
+
+    default_resource_root = os.path.join(getPackageDir("obs_fiberspectrograph"), "corrections")
+    """Default resource path root to use to locate header correction files."""
+
+    DETECTOR_MAX = 1
+
+    _const_map = {
+        "detector_num": 0,
+        "detector_name": "0",
+        "exposure_group": None,
+        "object": None,
+        "physical_filter": "empty",
+        "detector_serial": "0xdeadbeef",
+        "detector_group": "None",
+        "relative_humidity": None,
+        "pressure": None,
+        "temperature": None,
+        "focus_z": None,
+        "boresight_airmass": None,
+        "boresight_rotation_angle": None,
+        "tracking_radec": None,
+        "telescope": SIMONYI_TELESCOPE,
+        "observation_type": "spectrum",  # IMGTYPE is ''
+    }
+    """Constant mappings"""
+
+    _trivial_map = {
+        "observation_id": "OBSID",
```

Comment:
What fills in the OBSID? I don't see it listed in the https://github.com/lsst-ts/ts_fiberspectrograph/blob/develop/python/lsst/ts/fiberspectrograph/data_manager.py

The test file has the integer 21, which is not going to work unless 21 is an incrementing integer for every single exposure.

OBSID has to be unique for all observations that will ever be taken.

Reply:
As discussed on SLAC, there were changes to the CSC that now writes a "proper" OBSID.  This means that the field previously called OBSID (but really SEQNUM) changed.